common/persistence/nosql/nosqlplugin/types.go

```diff
@@ -90,10 +90,12 @@ type (
 	TransferTask = persistence.TransferTaskInfo
 
 	HistoryMigrationTask struct {
-		Transfer    *TransferTask
-		Timer       *TimerTask
-		Replication *ReplicationTask
-		Task        *persistence.DataBlob
+		Transfer      *TransferTask
+		Timer         *TimerTask
+		Replication   *ReplicationTask
```

Comment:
Could this struct be turned into generic struct?

Reply:
I don't bother to do that because it will be gone after the migration is done.